crates/ruff/src/checkers/logical_lines.rs

```diff
@@ -149,6 +150,18 @@ pub fn check_logical_lines(
             }
         }
 
+        if line.flags.contains(TokenFlags::BRACKET) {
+            for diagnostic in whitespace_before_parameters(
+                &line.tokens,
+                matches!(autofix, flags::Autofix::Enabled)
+                    && settings.rules.should_fix(&Rule::WhitespaceBeforeParameters),
```

Comment:
will probably move this check inside `whitespace_before_parameters` because the rule does not exist without `logical_fines` feature